.github/workflows/ci.yml

```diff
@@ -88,6 +89,7 @@ jobs:
         rm -rf ./ify-deps/vcpkg/buildtrees
         rm -rf ./ify-deps/vcpkg/packages
         rm -rf ./ify-deps/vcpkg/downloads
+        rm -rf ./ify-deps/vcpkg/installed/x64-windows-static/tools/qt5/debug
```

Comment:
This code should be called if the os is `macOS`

Reply:
Probably you should use this (I'm not sure) 
```suggestion
        rm -rf ./ify-deps/vcpkg/installed/x64-osx/tools/qt5/debug
```

Reply:
Thanks!